lib/command-context.coffee

```diff
@@ -29,6 +30,12 @@ class CommandContext
       runtime.didNotBuildArgs errorSendByArgs
       return false
 
+    if not runOptions.workingDirectory? or runOptions.workingDirectory is ''
+      # Precondition: lang? and lang of grammarMap
+      commandContext.workingDirectory = grammarMap[codeContext.lang][codeContext.argType].workingDirectory or ''
```

Comment:
@rgbkrk to be fully true.
if there will be a `''`, [this code](https://github.com/rgbkrk/atom-script/blob/master/lib/runner.coffee#L47) will return the 0'th path.